src/game/tower_defense/behaviors/target_ent.rs

```diff
@@ -0,0 +1,43 @@
+use bevy::prelude::*;
+use bevy_auto_plugin::auto_plugin::*;
+
+#[auto_register_type]
+#[auto_name]
```

Comment:
is this supposed to have name?

Reply:
Good point. copy and paste strikes as usual